Update 06/17/2022

SvicidalBug opened the pull request:
Small update to add a few Steam trading scams and fake movie websites.
- Removed `gametame.com`
  - This was mistakenly added in #64.
- Removed `vk.cc`
  - URL shortener used by [VKontakte](https://vk.com/).
- Removed `mediafiretrend.com`
  - Search engine for public files hosted on [MediaFire](https://mediafire.com/).

Commit:
Update 06/17/2022

Hexxium reviewed (approved):
Checked all domains:
tf2salute.com- unable to validate site authenticity, but its hidden register identity and recent registration makes it a likely scam site. Same case for rusty-games.com and rusty-slice.com and all the other clones in the list.

Checked the 3 removals, looks okay

Hexxium committed:
Update 6/18/2022

Adds one known steam message scam domain with proof: https://prnt.sc/A8_wCCsGnIB0

Hexxium committed:
Update 6/18/2022

Adds one domain

Hexxium committed:
Update 6/18/2022

Mainlist parity
